src/cljs/nr/utils.cljs

```diff
@@ -190,16 +191,24 @@
       (map (fn [[k v]] [(regex-of k) (span-of v)]))
       (sort-by (comp count str first) >))))
 
+; TODO duplicated for en... either take distinct of the result or figure out
+; how to get it in one map. the title/tr-title is a hot mess too
 (defn card-patterns-impl
   "A sequence of card pattern pairs consisting of a regex, used to match a card
   name in text, and the span fragment that should replace it"
   []
-  (letfn [(span-of [title] [:span {:class "fake-link" :data-card-title title} title])]
-    (->> (:all-cards-and-flips @app-state)
-         (vals)
-         (remove :replaced_by)
-         (map (fn [c] [(:title c) (span-of (:title c))]))
-         (sort-by (comp count str first) >))))
+  (letfn [(span-of [title tr-title] [:span {:class "fake-link" :data-card-title title} tr-title])]
+    (concat
+     (->> (:all-cards-and-flips @app-state)
+          (vals)
+          (remove :replaced_by)
+          (map (fn [c] [(:title c) (span-of (:title c) (:title c))]))
+          (sort-by (comp count str first) >))
+     (->> (:all-cards-and-flips @app-state)
+          (vals)
+          (remove :replaced_by)
+          (map (fn [c] [(tr-data :title c) (span-of (:title c) (tr-data :title c))]))
+          (sort-by (comp count str first) >)))))
```

Comment:
This isn't very good but I couldn't come up with anything nicer. Any suggestions would be welcome.